### DISCUSSION TITLE
Remove algorithmic feed

### DISCUSSION THREAD

**Discussion by wwahammy** [community/community#66155] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please remove the algorithmic feed. I don't want it.

**Comment by jorge07** [comment]
At least allow order by newest... Current one is literally useless, top on the list is someone followed me 2 months ago, which I can't care less...

**Comment by cotes2020** [comment]
The guy who came up with this _Feed_ should be fired :)

**Reply by wwahammy** [reply]
Let's be kind and focus on the experience, not the individuals.

**Reply by po5** [reply]
These decisions are taken by individuals actively trying to ruin tech and it's working.

**Comment by woheller69** [comment]
I don't want it either. Please revert

**Comment by ricardojoserf** [comment]
Please revert to the previous view :(

**Comment by msfjarvis** [comment]
This new feed is a MASSIVE regression in usability and should immediately be reverted back, I don't understand how something this broken is being shipped out to users.

**Comment by cj-mills** [comment]
The new home feed is worse than nothing at all. I tried using the filters to approximate the original setup, but the information layout is still worse. I'm technically getting less information but feeling overloaded because of the presentation.

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>
